src/main/resources/texts/texts.properties

```diff
@@ -0,0 +1,2 @@
+WELCOME: Hello, this is BankInfoBot, I can provide a lot of information about your banking transactions.\nHow can I adress you? 
+FUNCTIONS:  , what functions would you like to use?
```

Comment:
files with format `.properties` usually contain data like:
```
key1=value1
key.subkey=value2
```

Any reasons why you use `:` as separator?